test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
That's a good point, although I think we should just skip it in this case. Testing the docs code is always a sync-only smoke test. No matter how we slice this, it will leave some holes (or over-complicate the example, or over-complicate the code).